app/javascript/mastodon/features/emoji/database.ts

```diff
@@ -161,11 +161,23 @@ export async function putEmojiData(emojis: UnicodeEmojiData[], locale: Locale) {
   await trx.done;
 }
 
-export async function putCustomEmojiData(emojis: CustomEmojiData[]) {
+export async function putCustomEmojiData(
+  emojis: CustomEmojiData[],
+  clear = false,
```

Comment:
A named object parameter would make function calls using this option much clearer to read.